infra/travis/travis_build.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+import os
+import re
+import subprocess
+import sys
+import yaml
+
+
+DEFAULT_FUZZING_ENGINES = ['afl', 'libfuzzer']
+
+
+class HelperCommandException(Exception):
+  pass
+
+
+def get_modified_projects():
+  """Get a list of all the projects modified in this commit."""
+  output = subprocess.check_output(['git', 'diff', 'origin/master',
+                                    '--name-status'])
+  projects_regex = '.*projects/(?P<name>.*)/.*\n'
+  return re.findall(projects_regex, output)
+
+
+def get_oss_fuzz_root():
+  """Get the absolute path of the root of the oss-fuzz checkout."""
+  script_path = sys.argv[0]
+  return os.path.abspath(
+      os.path.dirname(os.path.dirname(os.path.dirname(script_path))))
+
+
+def execute_helper_command(helper_command):
+  """Execute |helper_command| using helper.py."""
+  root = get_oss_fuzz_root()
+  script_path = os.path.join(root, 'infra', 'helper.py')
+  command = ['python', script_path] + helper_command
+  process = subprocess.Popen(command)
+  process.wait()
+  if process.returncode != 0:
+    raise HelperCommandException('Command failed')
```

Comment:
nit: We can just use `subprocess.check_call(command)` here, and get rid of `HelperCommandException`.